src/ut_kvp.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2023 RDK Management
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+
+
+// ut_kvp.h, header to be expanded as required
+
+//#include <std_bool.h>
+//#include <std_int.h>
```

Comment:
std_int.h -> is required -> uint32_t is used.

I would prefer to change int to uint32_t I think so std_int is used everywhere.

std_bool.h gives you `bool` as a type